stories/useGravity.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { withKnobs, number } from '@storybook/addon-knobs';
+
+import { useGravity } from '../src';
+import './useGravity.css';
+
+export default {
+  title: 'Gravity',
+  decorators: [withKnobs],
+};
+
+export const GravityOpacity: React.FC = () => {
+  const [props] = useGravity<HTMLDivElement>({
+    from: { opacity: 0 },
+    to: { opacity: 1 },
+    config: {
+      moverMass: number('moverMass', 100000),
```

Comment:
The `number` piece is what configures this parameter as a knob that can get tweaked by a user running Storybook.